doc/manuals/user/filtering.md

```diff
@@ -142,7 +142,7 @@ operation filter.
 
```

Comment:
" doc/manuals/user/filtering.md 100644 → 100755"

It seems that file has change permissions from 644 to 755. I understand this is not on purpose so it should be changed back to 644.